rsciio/tiff/_api.py

```diff
@@ -352,15 +361,15 @@ def _read_serie(
     }
 
 
-def _load_data(serie, is_rgb, sl=None, memmap=None, **kwds):
-    dc = serie.asarray(out=memmap)
+def _load_data(handle, is_rgb, sl=None, memmap=None, **kwds):
+    dc = handle.asarray(out=memmap)
     _logger.debug("data shape: {0}".format(dc.shape))
     if is_rgb:
         from rsciio.utils import rgb_tools
 
         dc = rgb_tools.regular_array2rgbx(dc)
 
-    if sl is not None:
+    if sl is not None:  # TODO: What is sl; it is not covered.
```

Comment:
Looking at the history, it comes from https://github.com/hyperspy/hyperspy/commit/0e77a65e48b2bfb1a2f9a3f7f357b35873d454a4, and not used anymore since https://github.com/hyperspy/hyperspy/pull/2064/files.

It looks to me that this can be removed.